source/plugin/permissions/definitions.rst

```diff
@@ -0,0 +1,183 @@
+
+Allowed characters "A" - "Z", "a" - "z", "0" - "9", "_", "-" and ".".
+
+Examples
+~~~~~~~~
+
+* ``myPlugin``
+
+Grants **full** access to all plugin's permissions.
+
+* ``myPlugin.commands``
+
+Grants **full** access to all plugin's commands.
+
+* ``myPlugin.commands.teleport``
+
+Grants **full** access to the plugin's teleport command.
+
+* ``myPlugin.commands.teleport.execute``
+
+Only grants the user the permission to execute the command.
+(With this permission alone the user would only be able to teleport himself to others in his current world.)
+
+* ``myPlugin.commands.teleport.others``
+
+Only grants the user the permission to teleport others.
+(He still needs the ``myPlugin.commands.teleport.execute`` to actually execute the command.)
+
+* ``myPlugin.commands.teleport.all``
+
+Only grants the user the permission to teleport all players at once.
+(He still needs the ``myPlugin.commands.teleport.execute`` to actually execute the command.)
+
+* ``myPlugin.commands.teleport.worlds``
+
+Only grants the user the permission to teleport to all worlds.
+(He still needs the ``myPlugin.commands.teleport.execute`` to actually execute the command.)
+
+* ``myPlugin.commands.teleport.worlds.mynetherworld``
+
+Only grants the user the permission to teleport to *mynetherworld*.
+(He still needs the ``myPlugin.commands.teleport.execute`` to actually execute the command.)
+
+
+.. note::
+
+    There is no such thing as a ``myPlugin.commands.teleport.worlds.*`` wildcard permission.
+    Use ``myPlugin.commands.teleport.worlds`` for that.
+
+
+PermissionDescription
+=====================
+
+The permission description is an utility that is meant to provide the server owner with details on a certain permission.
+
+The description consists of:
+
+* the target permission id
+* a Text description
+* and an assigned role
+* the owning plugin
+
+If you have a dynamic element such as a world or item type then you can use ``<TemplateParts>``.
+
+Simple-Example
+~~~~~~~~~~~~~~
+
+.. code-block:: text
+    
+    myPlugin.commands.teleport.execute
+    
+    Description: Allows the user to execute the teleport command.
+    Role: user
+    Owner: MyPlugin v1.2.3
+
+Template-Example
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+    
+    myPlugin.commands.teleport.worlds.<World>
+    
+    Description: Allows the user to teleport to the world <World>.
+    Role: user
+    Owner: MyPlugin v1.2.3
+
+.. tip::
+
+    You might skip writing descriptions for some parent permission groups such as ``myPlugin.commands.teleport.worlds``
+    or ``myPlugin.commands`` as their meaning can be derived from the permission structure and the defined children
+    alone.
+
+Subject
+=======
+
+A Subject is a holder of assigned permissions it can be obtained from all CommandSources.
+Permissions can be granted or denied. Plugin authors will usually only deal with Subjects.
+
+Example
+~~~~~~~
+
+In the permission plugin's config this might look like this.
```

Comment:
This sentence is a bit clumsy. Drop the "In" and the first "this" and it makes more sense to me.